app/components/views/LNPage/PaymentsTab/PaymentsTab.module.css

```diff
@@ -112,7 +112,7 @@
   grid-template-columns: 1fr 3fr;
 }
 
-.paymentAccordion.vertical-accordion-arrow {
+.paymentAccordion .vertical-accordion-arrow {
```

Comment:
Hrmm didn't test that, but it looks problematic the way it's now.

This should be in camel case instead like all other classes, but also I think this is redundant and it
has to effect as it used now in PaymentsTab/Page.jsx:

```
  arrowClassName="vertical-accordion-arrow"
  className={styles.paymentAccordion}
```